### ISSUE TITLE
market buy order throwing error

### ISSUE THREAD

**Issue by adityamertia** [binance-exchange/node-binance-api#41]
Hi,

After upgrading from 0.3.0 to 0.3.6 or 0.3.7 the market order is failing with following error:
```
/home/ec2-user/etherlitecoinproject/node_modules/node-binance-api/node-binance-api.js:89
			if ( response && response.statusCode !== 200 ) throw response;                                               
[object Object]
```
Also updating in package.json to 0.3.0 and doing npm install doesn't move the core node-binance-api to 0.3.0 but keeps it at 0.3.7 and the error then changes to below:

`BUY(LTCETH,0.01,0)  { code: -1013, msg: 'Invalid price.' }`

These are breaking changes as existing code stops working.

**Comment by jaggedsoft** [comment]
v0.3.7 should have that breaking change removed, sorry about that. See
https://github.com/jaggedsoft/node-binance-api/issues/50

**Comment by adityamertia** [comment]
Ok. But on 0.3.7 i see below error on market order. market order has nothing to do with price?

`BUY(LTCETH,0.01,0) { code: -1013, msg: 'Invalid price.' }`

**Comment by jaggedsoft** [comment]
Resolved in `v0.3.8` https://github.com/jaggedsoft/node-binance-api/commit/8bf3cde02379e20ba085d9460d9158487515d8d7
